### PR TITLE
remove link to itself

### PR DESCRIPTION
**Pull request by dongxuwang**
This link probably shouldn't be list in the page itself

### CODE REVIEW

**Review by He-Pin — APPROVED** [review]
lgtm

File: docs/src/main/paradox/security/index.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ report with the Lightbend Akka team.
 
 ## Security Related Documentation
 
- * [Akka security fixes]($pekko.doc.dns$/docs/pekko/current/security/index.html)
```
Comment (pjfanning) [inline docs/src/main/paradox/security/index.md:22]: I think that this was meant to point to the Akka docs but was accidentally changed to be a Pekko link.

Could you keep this but change the link to https://akka.io/security/ ?

**Review by pjfanning — CHANGES_REQUESTED** [review]

File: docs/src/main/paradox/security/index.md (hunk 1 above)
Comment (He-Pin) [reply docs/src/main/paradox/security/index.md:22]: I see.

Comment (He-Pin) [reply docs/src/main/paradox/security/index.md:22]: image

it's `

Comment (dongxuwang) [reply docs/src/main/paradox/security/index.md:22]: updated, thank you

**Review by pjfanning — APPROVED** [review]
lgtm - thanks

**Comment by pjfanning** [comment]
@He-Pin are you happy to have this merged?

**Review by He-Pin — APPROVED** [review]
Lgtm

**Pull request merged**